### DISCUSSION TITLE
请教数据源设计的原理

### DISCUSSION THREAD

**Discussion by weichenzhiyi** [ly525/luban-h5#186] [Q&A]
请教数据源设计的原理。请问下：数据依赖主要配置什么？数据处理是处理回调函数或者返回数据吗？如果数据接口返回格式不一样，数据应如何渲染？

**Comment by ly525** [comment]
数据源的构成条件：
- name/key
- desc
- api url
- api method
- api request payload handler #用于生成 request payload
- api response handler # 用于处理 response


数据源的渲染和正常加载数据，然后挂载至 vuex 进行渲染一致
